buildcontext/excludes.go

```diff
@@ -23,7 +24,7 @@ var ImplicitExcludes = []string{
 	earthlyIgnoreFile,
 }
 
-func readExcludes(dir string, noImplicitIgnore bool) ([]string, error) {
+func readExcludes(dir string, noImplicitIgnore bool, excludeFromDockerIgnore bool) ([]string, error) {
```

Comment:
I think this would be more clear (and less chars)

```suggestion
func readExcludes(dir string, noImplicitIgnore bool, readDockerIgnore bool) ([]string, error) {
```

or alternatively `useDockerIngnore`.

Reply:
done